### PR TITLE
[10.0] [ADD] letsencrypt_transip_nl

### PR DESCRIPTION
**Pull request by janverb**
This is a continuation of #179. It depends on OCA/server-tools#1813.

Add a module for doing Let's Encrypt validation with TransIP's DNS service.

### CODE REVIEW

File: oca_dependencies.txt
Comment (hbrunn) [inline oca_dependencies.txt:4]: I'll keep this as reminder that we can't merge before our dependency is merged and this changed

**Review by hbrunn — CHANGES_REQUESTED** [review]

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Comment by NL66278** [comment]
Replaced by this one: squashed the fixup commits and no longer changes oca dependencies:

https://github.com/OCA/l10n-netherlands/pull/341

**Pull request closed without merging**
